charts/timed/values.yaml

```diff
@@ -221,6 +221,18 @@ postgresql:
   image:
     # postgresql.image.tag -- PostgreSQL image version to use
     tag: 12.2.0
+  ingress:
+    # postgresql.ingress.enabled -- Enable ingress
+    enabled: false
+    # postgresql.ingress.loadBalancerSourceRanges -- Whitelist specific IP ranges
+    loadBalancerSourceRanges: []
+    # postgresql.ingress.labels -- Service labels
```

Comment:
I would nitpick about these missing a `@default` annotation to ensure that the defaults don't result in a huge block of JSON inlined into the docs, but then I looked at the rest of the chart which doesn't care either.